src/mono/wasm/runtime-test.js

```diff
@@ -245,79 +286,46 @@ var Module = {
 			// for testing purposes add BCL assets to VFS until we special case File.Open
 			// to identify when an assembly from the BCL is being open and resolve it correctly.
 			/*
-			var content = new Uint8Array (read (asset, 'binary'));
-			var path = asset.substr(config.deploy_prefix.length);
-			writeContentToFile(content, path);
+			const content = new Uint8Array (read (asset, 'binary'));
+			const path = asset.substr(config.deploy_prefix.length);
+			IOHandler.writeContentToFile(content, path);
 			*/
-
-			if (typeof window != 'undefined') {
-				return fetch (asset, { credentials: 'same-origin' });
-			} else {
-				// The default mono_load_runtime_and_bcl defaults to using
-				// fetch to load the assets.  It also provides a way to set a
-				// fetch promise callback.
-				// Here we wrap the file read in a promise and fake a fetch response
-				// structure.
-				return new Promise ((resolve, reject) => {
-					var bytes = null, error = null;
-					try {
-						bytes = read (asset, 'binary');
-					} catch (exc) {
-						error = exc;
-					}
-					var response = { ok: (bytes && !error), url: asset,
-						arrayBuffer: function () {
-							return new Promise ((resolve2, reject2) => {
-								if (error)
-									reject2 (error);
-								else
-									resolve2 (new Uint8Array (bytes));
-						}
-					)}
-					}
-					resolve (response);
-				})
-			}
+			return IOHandler.fetch (asset, { credentials: 'same-origin' });
 		};
 
 		MONO.mono_load_runtime_and_bcl_args (config);
 	},
 };
 
-loadScript ("dotnet.js");
-
-const IGNORE_PARAM_COUNT = -1;
-
-var App = {
+const App = {
 	init: function () {
-		var wasm_set_main_args = Module.cwrap ('mono_wasm_set_main_args', 'void', ['number', 'number']);
-		var wasm_strdup = Module.cwrap ('mono_wasm_strdup', 'number', ['string']);
+		const wasm_set_main_args = Module.cwrap ('mono_wasm_set_main_args', 'void', ['number', 'number']);
+		const wasm_strdup = Module.cwrap ('mono_wasm_strdup', 'number', ['string']);
 
 		Module.wasm_exit = Module.cwrap ('mono_wasm_exit', 'void', ['number']);
 
 		console.info("Initializing.....");
 
-		for (var i = 0; i < profilers.length; ++i) {
-			var init = Module.cwrap ('mono_wasm_load_profiler_' + profilers [i], 'void', ['string'])
-
+		for (let i = 0; i < profilers.length; ++i) {
+			const init = Module.cwrap ('mono_wasm_load_profiler_' + profilers [i], 'void', ['string'])
```

Comment:
missing ; at end of line here, but it's harmless since the 'init()' will parse unambiguously

Reply:
Regardless, for consistency we should have semi colons that the end of every statement. I missed this one. Thanks.